### ISSUE TITLE
Search for tail number fails

### ISSUE THREAD

**Issue by 76rhodan** [fredclausen/docker-acarshub#52]
I'm running on ACARS Hub Version: v2.0 Build 1027 and it seems, that searching for tail numbers does not work as expected. I wanted to look for all messages originating from Tail A7-BFS. So I went to search-database -> search for mesages and c&p the tail number at the corresponding field.
That yielded no results - but there must be something, as I got an alert from this tail. Entering the tail manually into the field indicated, that the selector works until I enter the dash. That means, I get everything on A or A7 but nothing for A7-
There are no results for A7BFS as well but: A7 BFS does work (space instead of -).

**Comment by fredclausen** [comment]
Thanks for letting me know and I'll take a gander at it.

**Comment by fredclausen** [comment]
Unfortunately we don't see any dashes in callsigns in the US so I can't reproduce the issue. In order for me to diagnose the issue properly I'll need to see what the database is storing for those messages. If you could temporarily set

```yaml
      - QUIET_LOGS=
```

In your docker-compose.yaml file, and then provide the outputted JSON message it spits out for any aircraft that includes a dash in the tail field, that would be sufficient.

It will look something like

```console
[vdlm2dec-00012507] 2021/06/24 16:55:14 {"timestamp":1624553714.4660671,"station_id":"CS-KABQ-VDLM","channel":2,"freq":136.975,"icao":11390118,"toaddr":1057098,"is_response":0,"is_onground":0,"mode":"2","label":"H1","block_id":"0","ack":"!","tail":"N989AU","flight":"AA1621","msgno":"D16A","text":"#DFBA321,028578,1,1,TB000000/REP291,00,00,4/\r\nTRP KPHL KSAN  8 8\r\n/A1 164305, 35.5393,-104.8540,340,175.5,472,0.000064,\r\n0.000038, 42.12,- 94.30, 19.5,-38, 7.38,0\r\n/A2 164605, 35.4878,-105.2873,340,175.2,473,0.","end":true}
```

**Comment by 76rhodan** [comment]
Hi  Fred,

just fired it up and got the output below. Do you need more samples?

```acarshub    | [vdlm2dec] 2021/06/24 20:43:06 {"timestamp":1624567386.073802,"station_id":"76-EDDF1-VDLM","channel":0,"freq":136.725,"icao":3961030,"toaddr":1153050,"is_response":0,"is_onground":0,"mode":"2","label":"H1","block_id":"7","ack":"!","tail":"D-ALFF","flight":"LH8386","msgno":"D44G","text":"#DFB88 43076000061 8261\r\n   0)   0)  62 861  682179    0    0204105\r\n   0)   0)  66 886  674156    0    0\r\n50F06441020102000A1\r\n50F06441020102001B1\r\n00000000000010075\r\n00000000000010076\r\n 8275 8280 8335 84851  ","end":true}
acarshub    | [vdlm2dec] 2021/06/24 20:43:08 {"timestamp":1624567388.2227139,"station_id":"76-EDDF1-VDLM","channel":0,"freq":136.725,"icao":3961030,"toaddr":1153050,"is_response":0,"is_onground":0,"mode":"2","label":"H1","block_id":"8","ack":"!","tail":"D-ALFF","flight":"LH8386","msgno":"D44H","text":"#DFB60  855  680186\r\n 8290 8220 8300 83951  65  869  674158\r\n43600044908490001  3226D000FA8  2212 34  853\r\n43775044908490021  3196D000FA8  2237 37  851\r\nF5   34500  13 079507940794   672\r\n   531 2251867285 6419","end":true}
acarshub    | [webapp] 2021/06/24 20:43:09 [init]: starting scheduler
acarshub    | [webapp] 2021/06/24 20:43:09 [rrdtool]: Database found
acarshub    | [vdlm2dec] 2021/06/24 20:43:10 {"timestamp":1624567390.1706021,"station_id":"76-EDDF1-VDLM","channel":0,"freq":136.725,"icao":3961030,"toaddr":1153050,"is_response":0,"is_onground":0,"mode":"2","label":"H1","block_id":"9","ack":"!","tail":"D-ALFF","flight":"LH8386","msgno":"D44I","text":"#DFB0  293 -04 797\r\n   574 2241863284 70198  310 -05 806\r\n 880 880 8801100 849324574677  1  2\r\n 880 880 8801102 844724619676  2  2\r\n14424 000)44506 226 8925607  3  8340\r\n14400 000)44617 225 8895601  5  8296\r\n01","end":true}
acarshub    | [vdlm2dec] 2021/06/24 20:43:12 {"timestamp":1624567392.0685639,"station_id":"76-EDDF1-VDLM","channel":0,"freq":136.725,"icao":3961030,"toaddr":1153050,"is_response":0,"is_onground":0,"mode":"2","label":"H1","block_id":"0","ack":"!","tail":"D-ALFF","flight":"LH8386","msgno":"D44J","text":"#DFB3107038000 34135052000107 8490\r\n023045058000291 53073000058 8446\r\n  62  861  682179    0    0204141\r\n  66  886  674156    0    0204142"}```

**Comment by fredclausen** [comment]
Nope, that should do it!

**Comment by fredclausen** [comment]
Thanks for the bug report, and this should be fixed in 2.1

Until 2.1 is out you've found the work around - just replace the dash with a space. Other characters may introduce the same behavior, and all have the same work around.
